scripts/project-setup-scripts/google-project-owner.yaml

```diff
@@ -0,0 +1,1686 @@
+- servicedirectory.namespaces.list
+- servicedirectory.services.get
+- servicedirectory.services.getIamPolicy
+- servicedirectory.services.list
+- servicedirectory.services.resolve
+- servicemanagement.consumerSettings.get
+- servicemanagement.consumerSettings.list
+- servicemanagement.services.get
+- servicemanagement.services.list
+- servicenetworking.operations.get
+- servicenetworking.operations.list
+- servicenetworking.services.get
+- serviceusage.operations.get
+- serviceusage.operations.list
+- serviceusage.quotas.get
+- serviceusage.services.get
+- serviceusage.services.list
+- source.repos.get
+- source.repos.getIamPolicy
+- source.repos.list
+- spanner.backupOperations.get
+- spanner.backupOperations.list
+- spanner.backups.get
+- spanner.backups.getIamPolicy
+- spanner.backups.list
+- spanner.databaseOperations.get
+- spanner.databaseOperations.list
+- spanner.databases.beginReadOnlyTransaction
+- spanner.databases.get
+- spanner.databases.getDdl
+- spanner.databases.getIamPolicy
+- spanner.databases.list
+- spanner.databases.partitionQuery
+- spanner.databases.partitionRead
+- spanner.databases.read
+- spanner.databases.select
+- spanner.instanceConfigs.get
+- spanner.instanceConfigs.list
+- spanner.instanceOperations.get
+- spanner.instanceOperations.list
+- spanner.instances.get
+- spanner.instances.getIamPolicy
+- spanner.instances.list
+- spanner.sessions.create
+- spanner.sessions.delete
+- spanner.sessions.get
+- spanner.sessions.list
+- stackdriver.projects.get
+- storage.buckets.list
+- storage.hmacKeys.get
+- storage.hmacKeys.list
+- storagetransfer.jobs.get
+- storagetransfer.jobs.list
+- storagetransfer.operations.get
+- storagetransfer.operations.list
+- storagetransfer.projects.getServiceAccount
+- subscribewithgoogledeveloper.tools.get
+- tpu.acceleratortypes.get
+- tpu.acceleratortypes.list
+- tpu.locations.get
+- tpu.locations.list
+- tpu.nodes.get
+- tpu.nodes.list
+- tpu.operations.get
+- tpu.operations.list
+- tpu.tensorflowversions.get
+- tpu.tensorflowversions.list
+- trafficdirector.networks.getConfigs
+- transcoder.jobTemplates.get
+- transcoder.jobTemplates.list
+- transcoder.jobs.get
+- transcoder.jobs.list
+- vmmigration.deployments.get
+- vmmigration.deployments.list
+- vmwareengine.services.view
+- vpcaccess.connectors.get
+- vpcaccess.connectors.list
+- vpcaccess.connectors.use
+- vpcaccess.locations.list
+- vpcaccess.operations.get
+- vpcaccess.operations.list
+- workflows.executions.get
+- workflows.executions.list
+- workflows.locations.get
+- workflows.locations.list
+- workflows.operations.get
+- workflows.operations.list
+- workflows.workflows.get
+- workflows.workflows.getIamPolicy
+- workflows.workflows.list
+# </viewer> End of permissions included as part of roles/viewer
+# Add permissions for the other roles we need:
+##roles/billing.projectManager (updated 2/10/2021)
+- resourcemanager.projects.createBillingAssignment
+- resourcemanager.projects.deleteBillingAssignment
```

Comment:
I think we want to remove these based on the fact that users will no longer be setting billing accounts on projects and Terra will handle that for users